modules/300-prometheus/templates/prometheus/alert-relable-secret.yaml

```diff
@@ -0,0 +1,22 @@
+{{- define "additional_relable_config_tier" }}
+{{- range $key, $value := .Values.prometheus.externalLabels }}
+- sourceLabels: [{{ $key }}]
+  targetLabel: __tmp_{{ $key }}_exists
+- sourceLabels: [__tmp_{{ $key }}_exists]
+  regex: "^$"
+  targetLabel: {{ $key }}
+  replacement: {{ $value | quote }}
+- target_label: __tmp_{{ $key }}_exists
```

Comment:
Do you know if we need this relabel? According to the Prometheus documentation, all labels prefixed with double underscore are removed automatically.